central/cluster/util/util.go

```diff
@@ -0,0 +1,42 @@
+package util
+
+import (
+	"context"
+	"errors"
+
+	cluster "github.com/stackrox/rox/central/cluster/datastore"
+)
+
+var (
+	// ErrClusterNotFound indicates a cluster was not found.
+	ErrClusterNotFound = errors.New("cluster not found")
```

Comment:
Let's use our [roxerrors](https://github.com/stackrox/stackrox/blob/master/pkg/errox/errors.go#L9) for that one (then you can skip defining it here).

Reply:
https://github.com/stackrox/stackrox/pull/7738/commits/177eeddbf27caa0c7f50eb031220982b05b04fe3